mlflow/utils/validation.py

```diff
@@ -148,6 +149,30 @@ def _validate_model_version_tag(key, value):
     _validate_length_limit("Model version value", MAX_MODEL_REGISTRY_TAG_VALUE_LENGTH, value)
 
 
+def _validate_experiment_pagination(max_results):
```

Comment:
change the fn name to reflect what it's actually checking:
_validate_experiment_pagination() -> _validate_max_results()
Or if you want to validate more stuff later, _validate_list_experiments_input() is a good choice.

Reply:
Makes sense. I'll fix it.

Reply:
Makes sense. Renamed this function.